code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -38,7 +38,8 @@
 		if(prob(98))
 			randmutb(M)
 		else
-			randmutg(M)
+			var/datum/mutation/human/HM = pick((good_mutations) - mutations_list[HULK])
+			HM.force_give(M)
```

Comment:
this is already the same as what the proc does except it rolls every time instead of sometimes not rolling if it picked hulk